README.md

```diff
@@ -48,22 +48,38 @@ Before you attempt to make a contribution please read the [Community Participati
 
 ## Build Instructions
 
+Note: Both Android SDK and NDK are required.
 
 1. Clone or Download the repository:
 
   ```shell
   git clone https://github.com/mozilla-mobile/fenix
   ```
 
-2. Import the project into Android Studio **or** build on the command line:
+2. **Import** the project into Android Studio **or** build on the command line:
 
   ```shell
-  ./gradlew clean app:assembleArmDebug
+  ./gradlew clean app:assembleGeckoBetaDebug
   ```
+  
+  Use app:assembleGeckoNightlyDebug to build with the Gecko Nightly version instead.
 
-3. Make sure to select the correct build variant in Android Studio:
-**armDebug** for ARM
-**x86Debug** for X86
+3. Make sure to select the correct build variant in Android Studio. See the next section.
```

Comment:
I might also suggest adding the recommended flavor in this line too (if people don't want to read the next section).
build variant in Android Studio (recommended: **geckoBetaDebug**).